vendor/bat-native-ads/BUILD.gn

```diff
@@ -573,6 +573,7 @@ source_set("ads") {
 
   deps = [
     "//base",
+    "//brave/common",
```

Comment:
This is a layering violation and is not allowed here, nor is it allowed in components/brave_ads